### ISSUE TITLE
[BUG] - Uncaught TypeError: Cannot read properties of undefined (reading 'AbortError') on web

### ISSUE THREAD

**Issue by dtfiedler** [warp-contracts/warp#519]
**Describe the bug**
When using `warp-contracts@^1.4.26` in the browser we get the following error

```
Uncaught TypeError: Cannot read properties of undefined (reading 'AbortError')
```

`AbortError` was introduced in this change - https://github.com/warp-contracts/warp/compare/v1.4.26...v1.4.27

**To Reproduce**
1. Standup a simple react app
2. Install `warp-contracts@1.4.39`
3. Run the application

**Expected behavior**
Warp properly exports mjs/esm modules for enviornments

**Desktop (please complete the following information):**
 - OS: MaxOS
 - Node.js version - 18.17.1
 - warp-contracts SDK version - 1.4.39
 - warp-contracts plugins being used - None
 - Browser [e.g. chrome, safari] - Chrome

**Additional Context**
It looks like the bundle file being produced includes some `/mjs` imports that are causing the issue

```
// node_modules/warp-contracts/lib/mjs/index.js
init_shim();
var import_cjs = __toESM(require_cjs2());
var mjs_default = import_cjs.default;
var AbortError = import_cjs.default.AbortError;
var AbstractContractHandler = import_cjs.default.AbstractContractHandler;
var ArweaveContractDefinitionLoader = import_cjs.default.ArweaveContractDefinitionLoader;
var ArweaveGatewayBundledContractDefinitionLoader = import_cjs.default.ArweaveGatewayBundledContractDefinitionLoader;
var ArweaveGatewayBundledInteractionLoader = import_cjs.default.ArweaveGatewayBundledInteractionLoader;
var ArweaveGatewayInteractionsLoader = import_cjs.default.ArweaveGatewayInteractionsLoader;
var ArweaveWrapper = import_cjs.default.ArweaveWrapper;
var Benchmark = import_cjs.default.Benchmark;
var CacheKey = import_cjs.default.CacheKey;
var CacheableInteractionsLoader = import_cjs.default.CacheableInteractionsLoader;
var CacheableStateEvaluator = import_cjs.default.CacheableStateEvaluator;
var ConsoleLogger = import_cjs.default.ConsoleLogger;
var ConsoleLoggerFactory = import_cjs.default.ConsoleLoggerFactory;
var ContractCache = import_cjs.default.ContractCache;
var ContractCallRecord = import_cjs.default.ContractCallRecord;
var ContractError = import_cjs.default.ContractError;
var ContractMetadata = import_cjs.default.ContractMetadata;
var CustomEvent = import_cjs.default.CustomEvent;
var DEFAULT_LEVEL_DB_LOCATION = import_cjs.default.DEFAULT_LEVEL_DB_LOCATION;
var DefaultEvaluationOptions = import_cjs.default.DefaultEvaluationOptions;
var DefaultStateEvaluator = import_cjs.default.DefaultStateEvaluator;
var EvalStateResult = import_cjs.default.EvalStateResult;
var EvaluationOptionsEvaluator = import_cjs.default.EvaluationOptionsEvaluator;
var HandlerBasedContract = import_cjs.default.HandlerBasedContract;
var HandlerExecutorFactory = import_cjs.default.HandlerExecutorFactory;
var InnerWritesEvaluator = import_cjs.default.InnerWritesEvaluator;
var InteractionCall = import_cjs.default.InteractionCall;
var InteractionCompleteEvent = import_cjs.default.InteractionCompleteEvent;
var InteractionInput = import_cjs.default.InteractionInput;
var InteractionOutput = import_cjs.default.InteractionOutput;
var JsHandlerApi = import_cjs.default.JsHandlerApi;
var KV = import_cjs.default.KV;
var KnownErrors = import_cjs.default.KnownErrors;
var LevelDbCache = import_cjs.default.LevelDbCache;
```

**Comment by foxoffire33** [comment]
'm experiencing the same issue. I'm using Vite and trying to utilize ESM modules.

**Comment by ppedziwiatr** [comment]
Hey @dtfiedler can you please create some repo with a MRE? or at least please let me know how exactly you're importing the warp sdk - but the repo would be preferred :) thanks!

**Comment by dtfiedler** [comment]
An example of the issue can be found on this branch - https://github.com/ar-io/ar-io-sdk/blob/alpha/examples/vite/src/App.tsx#L8-L40

Some details:
- we're using `vite` to package the web app
- the `tsconfig.json` is using `moduleResultion: node` (here)[https://github.com/ar-io/ar-io-sdk/blob/alpha/examples/vite/tsconfig.json#L12-L13] which should by default use `warp-contracts` cjs import [here]((https://github.com/warp-contracts/warp/blob/main/package.json#L6)
- we provide `nodePolyfills` to provide polyfills - although don't think it's necessary for the current setup

Interesting enough - it works fine with the webpack example [here](https://github.com/ar-io/ar-io-sdk/tree/alpha/examples/webpack)

**Comment by dtfiedler** [comment]
AH - looks like it's as simple as adding these export paths to the package.json - vite must do a bit more work than webpack when analyzing dependencies and their imports - https://github.com/warp-contracts/warp/pull/521

**Comment by ppedziwiatr** [comment]
Thanks, released in 1.4.42
